src/main/java/com/simon816/sponge/bootstrap/Bootstrap.java

```diff
@@ -84,7 +84,8 @@ private static void findAndLoadJars() {
             @Override
             public boolean accept(File pathname) {
                 String fn = pathname.getName().toLowerCase();
-                return fn.endsWith(".jar") && fn.contains("forge") && fn.contains("-universal") && supportedVersion(fn);
+                return fn.endsWith(".jar") && (fn.contains("forge") || fn.contains("FTBserver")) 

```

Comment:
gah. FTBserver here needs to be lower case, forgot I copied my checkout elsewhere on my machine